<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorList</class>
 <widget class="QDockWidget" name="ErrorList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Error List</string>
  </property>
  <widget class="QWidget" name="errorListContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QToolBar" name="toolBar">
      <property name="windowTitle">
       <string>toolBar</string>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextBesideIcon</enum>
      </property>
      <addaction name="actionErrors"/>
      <addaction name="actionWarnings"/>
      <addaction name="actionInformation"/>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="list">
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerMinimumSectionSize">
       <number>10</number>
      </attribute>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>X</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Message</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionErrors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Errors</string>
   </property>
  </action>
  <action name="actionWarnings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Warnings</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
